cg/seeds/example_data_attributes_config.js

```diff
@@ -0,0 +1,27 @@
+exports.seed = async function(knex, Promise) {
```

Comment:
What will this seed be used for?

Reply:
For creating a default configuration in the Controller side, will be used mainly for demo and elements sdk

Reply:
Oks